.travis.yml

```diff
@@ -6,7 +6,7 @@ php:
   - hhvm
 services: memcached
 before_script:
-  - bash -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-add .travisphpconfig.ini ; fi;'
+  - bash -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-add .travisphpconfig.ini ; echo no | pecl install -f apcu-4.0.10; fi;'
```

Comment:
What do we need to respond `no` when installing `apcu`?  Curious mostly, because usually it's `-y` or something like that to say yes to everything.